src/prefect/settings.py

```diff
@@ -525,6 +544,14 @@ def default_cloud_ui_url(settings, value):
 can affect retry lengths.
 """
 
+PREFECT_CLIENT_RETRY_EXTRA_CODES = Setting(
```

Comment:
Hm should we do extra codes or just `CODES` and have it default to our current list?

Reply:
I was thinking the same thing at first, but then I worry users might accidentally override the safe defaults. On the other hand, it's more optionality.

Reply:
Yeah I'd be a little concerned about that too. And like.. who doesn't want to retry on a 429?

Reply:
Exactly! I think it's reasonable to assume that users will always want to retry 429s and 503s.